sphinx/source/spelling_wordlist.txt

```diff
@@ -21,10 +21,11 @@ Numpy
 PhD
 Pieterse
 placeholder
+radians
+Roux
```

Comment:
Why don't you get the honour of having your surname?

Reply:
Yup. Spell check does not like your surname but is happy with mine